man/man8/zfs-create.8

```diff
@@ -234,14 +234,12 @@ if the volume is not sparse.
 Print verbose information about the created dataset.
 .El
 .El
-.Ss ZFS Volumes as Swap
-ZFS volumes may be used as swap devices.
-After creating the volume with the
-.Nm zfs Cm create Fl V
-enable the swap area using the
-.Xr swapon 8
-command.
-Swapping to files on ZFS filesystems is not supported.
+.Ss ZFS for Swap
+Swapping to a ZFS volume is possible, however care must be taken to avoid
+deadlocks.
```

Comment:
Maybe (A):

```suggestion
Swapping to a ZFS volume is possible, however care must be taken to minimise
the risk of deadlock.
```

Alternative (B):

> … the likelihood of deadlock.

Alternative (C):

> Swapping to a ZFS volume is prone to deadlock. OpenZFS FAQ includes an approach that may reduce the risk.

– and remove the subsequent line about FAQ.

Reply:
I like alternative C but I'd suggest phrasing it even more strongly.  How about just:

"Swapping to a ZFS volume is prone to deadlock and not recommended."

Reply:
Thanks. Done: 

* https://github.com/grahamperrin/zfs/commit/d8a86c5158bf2a8bc8a632bcddcc879cc62381e6